src/plugins/dashboard/public/dashboard_container/component/grid/dashboard_grid.tsx

```diff
@@ -125,8 +125,7 @@ export const DashboardGrid = ({ viewportWidth }: { viewportWidth: number }) => {
       className={classes}
       width={viewportWidth}
       breakpoints={breakpoints}
-      onDragStop={onLayoutChange}
-      onResizeStop={onLayoutChange}
+      onLayoutChange={onLayoutChange}
```

Comment:
This change fixes https://github.com/elastic/kibana/issues/165909. I fixed that in this PR because the problem is _much_ more obvious here.